packages/graphback-codegen-schema/src/definitions/schemaDefinitions.ts

```diff
@@ -258,8 +281,64 @@ export const buildMutationInputType = (modelType: GraphQLObjectType) => {
         return fieldObj;
       }, {})
   });
+
+  schemaComposer.add(mutationInputObject)
+}
+
+function mapObjectInputFields(schemaComposer: SchemaComposer<any>, fields: GraphQLField<any, any>[]): GraphQLInputField[] {
+  return fields.map((field: GraphQLField<any, any>) => {
+    let namedType = getNamedType(field.type)
+    let typeName = namedType.name
+
+    let inputType
+    if (isObjectType(namedType)) {
+      typeName = getInputTypeName(typeName, GraphbackOperationType.CREATE)
+      namedType = schemaComposer.getOrCreateITC(typeName).getType()
+
+      inputType = copyWrappingType(field.type, namedType)
```

Comment:
Why is copyingWrapping type needed?

Reply:
```
"""@model"""
type Note {
  id: ID!
  comments: [Comment]!
}
```

`Note.comments` will need to map to a newly created input type, so this ensures that the required level is the exact same in `CreateNoteInput.comments` as `Note.comments`.

```
type CreateNoteInput {
  id: ID!
  comments: [CreateCommentInput]!
}
```

Reply:
Gotcha.